webapp/client/src/components/AccordionComponent.js

```diff
@@ -90,6 +107,9 @@ export default function AccordionComponent({ title, intro, items, variant }) {
                 onClick={() => {
                   if (toggle === index) setToggle(-1);
                   else setToggle(index);
+                  addPlausibleGoal(plausibleDomain, `${item.title}`, {
+                    method: `Hilfebereich /${item.title}`,
```

Comment:
`Hilfebereich /${item.title}` the space after Hilfebreich should be removed or?

Reply:
yes it should. but there should also be a space after the slash. Thanks for that. I will adjust this and check the other methods.